packages/generator/pyproject.toml

```diff
@@ -24,6 +24,7 @@ Mako = "^1.2.1"
 click = ">=8.1.3"
 grpcio = "^1.49.1"
 protobuf = "^4.21"
+black = ">=24.8.0"
```

Comment:
why it is added as `>=` instead of `^`

Reply:
`>=` is used because any higher versions than 24.8.0 can be used but "^" only supports version of 24.x.x but not 25.

Reply:
`>=` is used because any higher versions than 24.8.0 can be used but "^" only supports version of 24.x.x but not 25.